tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
-
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+envlist = py27, coverage
 
+[testenv]
+deps = -rrequirements/tests.txt
 commands =
     pylama mozdownload
-    py.test {posargs}
```

Comment:
I would recommend two environments named `pylama` or the more generic `lint`, and `tests`. The `tests` environment would run the tests, gather coverage, and report to coveralls. The `pylama` or `lint` environment would just run the linting. Then we can change Travis to run the test environment on Linux and OSX, and the lint environment only on Linux.